apps/issuance/src/issuance.service.ts

```diff
@@ -313,4 +316,54 @@ export class IssuanceService {
       throw error;
     }
   }
+
+  async exportSchemaToCSV(credentialDefinitionId: string): Promise<object> {
+    try {
+      const schemaResponse: SchemaDetails = await this.issuanceRepository.getCredentialDefinitionDetails(credentialDefinitionId);
+
+      const jsonData = [];
+      const attributesArray = JSON.parse(schemaResponse.attributes);
+
+      // Extract the 'attributeName' values from the objects and store them in an array
+      const attributeNameArray = attributesArray.map(attribute => attribute.attributeName);
+      attributeNameArray.unshift('email');
+
+      const [csvData, csvFields] = [jsonData, attributeNameArray];
+
+      if (!csvData || !csvFields) {
+        // eslint-disable-next-line prefer-promise-reject-errors
+        return Promise.reject('Unable to transform schema data for CSV.');
+      }
+
+      const csv = parse(csvFields, { fields: csvFields });
+
+      const filePath = join(process.cwd(), `uploadedFiles/exports`);
+
+
+      let processedFileName: string = credentialDefinitionId;
+      processedFileName = processedFileName.replace(/[\/:*?"<>|]/g, '_');
```

Comment:
can we direct use this login on credentialDefinitionId processedFileName.replace(/[\/:*?"<>|]/g, '_')